models/eos/ospfv3/eos_ospfv3.yaml

```diff
@@ -0,0 +1,405 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+  {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+  }
+NETWORK_OS: eos
+RESOURCE: ospfv3
+COPYRIGHT: Copyright 2020 Red Hat
+
+DOCUMENTATION: |
+module: eos_ospfv3
+short_description: OSPFv3 resource module
+description: This module configures and manages the attributes of ospfv3 on Arista
+  EOS platforms.
+version_added: 1.0.0
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+notes:
+- Tested against Arista EOS 4.23.0F
+- This module works with connection C(network_cli). See the L(EOS Platform Options,../network/user_guide/platform_eos.html).
```

Comment:
```suggestion
- This module works with connection C(network_cli). See the R(EOS Platform Options,eos_platform_options).
```

Reply:
We can't use L() anymore for linking to Ansible docs.  See https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation for details.